public/locations-list-map.js

```diff
@@ -414,11 +645,18 @@ function centerMapToBounds(map, bounds, maxZoom) {
   }
 }
 
+// TODO (patricknelson: The code inside the .getCurrentPosition() is now DUPLICATED in centerMapToMarkersNearUser() AND centerMapToMarkersNearCoords()
+//  This is to prevent conflicts. I propose we completely scrap this function because
+//  1.) It is not properly named, since it's purpose is to center map on the USER coordinates (an API that isn't always available due to permissions)
+//  2.) It does too much; it centers map on user AND it automatically calculates bounds on nearby markers. However, that
+//      can be abstracted to normalized plain lat/lng coordinates instead.
+//  Therefore, I propose we completely replace this with a simple function call to centerMapToMarkersNearUser().
+
 function centerMapToNearestMarkers(map, markers, fallbackBounds) {
   // First check to see if the user will accept getting their location, if not, silently return
   if (navigator.geolocation) {
     navigator.geolocation.getCurrentPosition(
-      function (position) {
+      (position) => {
```

Comment:
@awong-dev this was using old `function()` syntax, converted these as well per your nits for consistency purposes.

Reply:
thanks.  Appreciate it. Given there's an actual semantic difference for `this`, it's more than just style so it's a good change.